Patches.py

```diff
@@ -2014,6 +2076,27 @@ def set_jabu_stone_actor(rom, actor_id, actor, scene):
     get_actor_list(rom, set_jabu_stone_actor)
 
 
+def set_spirit_shortcut_actors(rom):
+    def set_spirit_shortcut(rom, actor_id, actor, scene):
+        # Hack to swap silver block position in the vanilla shortcut hallway
+        # to allow movement from lobby -> statue room. If you want to use this,
+        # only set the flag for the east block pushed, then use this to move
+        # the north block to the other side of the hole.
+        #if actor_id == 0x00FF and scene == 6: # push blocks in spirit temple
```

Comment:
Dead code? The comment says "if you want to use this" but idk if it should be here just for people editing their own copies of the repo.

Reply:
With the logic changes made to spirit, this would make unbeatable seeds. It shouldn't be here anymore.

Original implementation moved one of the silver blocks to the other side of the whole in vanilla Spirit to maintain the silver gauntlets requirement to enter as adult. After discussion this was reverted in favor of logic changes and per-dungeon settings.